packages/docs/src/matchers/symmetric/index.ts

```diff
@@ -0,0 +1,63 @@
+import { Function } from "../../interfaces";
+
+import * as _toBe from "./toBe.json";
```

Comment:
We can just do `import blah from "blah.json";`

Reply:
Tried this, and for the Docusaurus site it returns undefined for each import. Apparently this is an issue with webpack.

Reply:
Assign me to the PR once you've pushed markups for everything else and I'll take a look

Reply:
Docusaurus handles this fine, both when building a release bundle and using hot reload